Bottom: 9ce160a2af08ef878c2aa540bf0aed1ef3f4531d
Top:    9063622604564df0a8cf83f0df4a0d21acdd373a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 01:08:26 +0000

hid/common: Fix nogui_confirm_dialog()


---

diff --git a/src/main.c b/src/main.c
index 923e0ee..6eaf438 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1010,6 +1010,8 @@ main (int argc, char *argv[])
   InitBuffers ();
   SetMode (ARROW_MODE);
 
+  gui->confirm_dialog ("This is a test");
+
   if (command_line_pcb)
     {
       /* keep filename even if initial load command failed;
